Source/WebKitLegacy/mac/WebCoreSupport/WebInspectorClient.mm

```diff
@@ -559,9 +559,7 @@ - (NSWindow *)window
     [window setMinFullScreenContentSize:NSMakeSize(minimumFullScreenWidth, minimumWindowHeight)];
     [window setCollectionBehavior:([window collectionBehavior] | NSWindowCollectionBehaviorFullScreenAllowsTiling)];
 
-#if HAVE(STAGE_MANAGER_NS_WINDOW_COLLECTION_BEHAVIORS)
     [window setCollectionBehavior:([window collectionBehavior] | NSWindowCollectionBehaviorAuxiliary)];
```

Comment:
This and the line above it can be combined as:

```
[window setCollectionBehavior:([window collectionBehavior] | NSWindowCollectionBehaviorFullScreenAllowsTiling | NSWindowCollectionBehaviorAuxiliary)];
```